client/me/privacy/controller.js

```diff
@@ -0,0 +1,19 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import userSettings from 'lib/user-settings';
```

Comment:
Huh, I suppose this was never ported to the data-layer? Note that if we need to update a user's preference vs settings, preferences are available from the global redux store.